CalendarFXView/src/main/java/com/calendarfx/model/EditOperation.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
```

Comment:
This enum is not part of the model. It is purely used by the view. Place enum inside DateControl next to entryEditPolicy property inside DateControl.